### ISSUE TITLE
emmm

### ISSUE THREAD

**Issue by mandyzore** [BetsyHJ/KSR#1]
Hey bro your code has some intent style and python version unification problems. It'll be helpful if u can fix them as soon as possible. Awsome paper by the way.

**Comment by MrRace** [comment]
Yeah, I find some aligning problems in the codes and just fix them one by one..........

**Comment by yufengwhy** [comment]
I also met the intent style problem... Could you share the fixed codes to your fork? @mandyzore @MrRace

**Comment by MrRace** [comment]
> I also met the intent style problem... Could you share the fixed codes to your fork? @mandyzore @MrRace

The core problem is how to get `File_ItemEmbedding，File_KBItemEmbedding和File_r_matrix`. It seems no reply for long time .........looking forward it
